### DISCUSSION TITLE
Direct links to Vault Items

### DISCUSSION THREAD

**Discussion by laxxie** [dani-garcia/vaultwarden#2546] [Ideas]
Bitwarden published the feature that each vault item has a unique idea which is shown now so you are able to copy it.

https://bitwarden.com/blog/link-vault-item-urls-for-quick-lookup-and-sharing/

Did you already had a chance to check this function and are you able to implement it. That would be a great improvement
(i know this stuff comes up every x times but now after Bitwarden has this feature...)

**Comment by BlackDex** [comment]
I haven't tested it yet, but this is all client side it looks like, and thus this should be working in the current `testing` tagged docker image.
